models/user.model.js

```diff
@@ -1,13 +1,14 @@
 const mongoose = require("mongoose");
 const { Schema } = mongoose;
 
+
 const UserSchema = new mongoose.Schema({
-  name : {type : String } ,
-  password : {type : String},
-  wishlist : [{type: mongoose.Schema.Types.ObjectId,ref: "Product"}],
-  cart : [    
-      {type: mongoose.Schema.Types.ObjectId,ref: "Product"}        
-    ]
+  name: { type: String },
+  password: { type: String },
```

Comment:
Here you can add a required field and a corresponding message, as one cannot proceed without entering them. Also, you can add a validator for password field.